scss/_o-carousel.scss

```diff
@@ -28,7 +29,7 @@
   transition: width .5s ease;
 
   &:focus {
-    outline: 1px dotted #000;
+    outline: 2px dotted #000;
```

Comment:
don't use color code in scss file use var instead currently you could use $carousel-control-color who's coherent or use btn styles ($input-focus-width and $input-focus-border-color)